k8s/charts/openebs/templates/deployment-maya-apiserver.yaml

```diff
@@ -7,19 +7,21 @@ metadata:
     chart: {{ template "openebs.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-    component: apiserver
+    component: maya-apiserver
```

Comment:
@vharsh - `component` is being used in label-selector below. Will this change from `apiserver` to `maya-apiserver` name cause any upgrade issues?

Reply:
It should cause one, I remember @amitd saying something similar.
We can verify this by checking the **before** and **after** state of @prateekpandey14 's change to the kubernetes/charts repository